pkg/applier/manager.go

```diff
@@ -151,7 +150,7 @@ func (m *Manager) runWatchers(ctx context.Context) {
 				if dir.IsDirectory(event.Name) {
 					m.createStack(ctx, stacks, event.Name)
```

Comment:
In the Windows unit test logs, I see paths in the logs like this: `stack="C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\TestManager_Rename2408267215\\001/lorem". Note the last slash, which should IMO be a backslash. IIUC the logged value is coming from fsnotify, so something is not normalizing path names there. Note that the slash is a valid separator on Windows, too, albeit not the canonical one.